app/assets/javascripts/components/account_page.es6.jsx

```diff
@@ -0,0 +1,55 @@
+class AccountPage extends Component {
+
+  static get defaultState() {
+    return { sidebar: true };
+  }
```

Comment:
Woo nice use of state here. We will probably abstract this out later on (since multiple pages have this sidebar functionality but good job!).